

// Basic
try {

    console.log(`Value of x is ${x}`)
}
catch(error) {
    console.log(error.message, ", Error")
}
finally {
    console.log("statement executed successfully!")
}

console.log()

// Using throw

// try {

//     if (x > 5) throw "Value of x must be lesser than or equal to 5"
//     console.log(`Value of x is ${x}`)
// }
// catch(error) {
//     console.log(error, ", Error")
// }
// finally {
//     console.log("statement executed successfully!")
// }

// function Add(a, b) {
//     return a + b + 10 + c
// }


// function Add(c, d) {
//     return a + b
// }

// let a = 10

// a = 6556